MarsRoverDisc.py

```diff
@@ -0,0 +1,133 @@
+from typing import Optional, Tuple
```

Comment:
What is this import from actually?
If it's not used, could you help to remove it?

Reply:
dk from where one, deleted